src/JMEJetAnalyzer.cc

```diff
@@ -145,10 +149,12 @@ void JMEJetAnalyzer::analyze(const edm::Event& iEvent,
          refdrjt.push_back(reco::deltaR(jet, *ref));
          refpdgid.push_back(ref->pdgId());
          refarea.push_back(ref->jetArea());
+         isMatched.push_back(true);
```

Comment:
This is already stored inside the tree, no need to duplicate it. See ``has_gen_match``